permissions/plugin-thinBackup.yml

```diff
@@ -2,6 +2,8 @@
 name: "thinBackup"
 github: "jenkinsci/thin-backup-plugin"
 paths:
+- "org/jenkins-ci/plugins/thin-backup-plugin"
```

Comment:
Are you looking at changing the groupId and artifactId for the plugin? The path is the groupId from the pom.xml with periods replaced with slashes and then the artifactId. If you change these, the plugin will basically become a completely different plugin and both the older version and the newer version can be installed at the same time, which is not a good idea.

Reply:
Oops! I reverted that